src/engine/core/string.h

```diff
@@ -84,7 +84,7 @@ template <int size> struct StaticString
 		return *this;
 	}
 
-	template <int size> void add(StaticString<size>& value) { Lumix::catString(data, size, value.data); }
+	template <int value_size> void add(StaticString<value_size>& value) { Lumix::catString(data, size, value.data); }
```

Comment:
template parameter hides outer template parameter.

BTW: seems like we need to pass `strlen(data)` as second argument to `catString`